src/cartesian/Area.tsx

```diff
@@ -738,14 +738,9 @@ function AreaImpl(props: WithIdRequired<Props>) {
 
   const { height, width, x: left, y: top } = plotArea;
 
-  /*
-   * It is important to NOT have this condition here,
```

Comment:
What's changed?

Reply:
`useAnimationId` now handles the state reset (that used to be done with the undefined tick). I tested in storybook too.